### ISSUE TITLE
Thumbnails for Downloaded Files

### ISSUE THREAD

**Issue by mzavada** [MicrosoftDocs/OfficeDocs-SharePoint#453]
Thumbnails do not render even for files and folders that are marked "Always keep on device."  Only when you open the file does the thumbnail appear.  This makes it difficult to work with large amounts of files when you need to see the thumbnails

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8f0d9527-a78b-d18b-d218-881fac11423d
* Version Independent ID: 3ad8fc7b-4869-dbdc-802b-a804f65d0e39
* Content: [Configure syncing with the new OneDrive sync client for Sharepoint 2019](https://docs.microsoft.com/en-us/sharepoint/install/new-onedrive-sync-client)
* Content Source: [SharePoint/SharePointServer/install/new-onedrive-sync-client.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointServer/install/new-onedrive-sync-client.md)
* Product: **sharepoint-server-itpro**
* GitHub Login: @kaarins
* Microsoft Alias: **kaarins**

**Comment by tseward** [comment]
Yep, I see the same behavior only against SharePoint 2019. ODfB Online/OneDrive personal appear to work like you want them to.

**Comment by mzavada** [comment]
Sorry I should have included the details of our setup.  This is with O365/SharePoint along with Windows 10 (16299.785) and OneDrive (18.212.1021.0007)

**Comment by tseward** [comment]
@mzavada I let the PG know in December of this issue -- they're aware of it. I have no ETA on a fix or what the priority of the fix is (or it may have been fixed already).

@scanum please close.
